### PR TITLE
Rhsm: new lens to parse subscription-manager's rhsm.conf

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: lenses/rhsm.aug [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  This lens applies to:
+    /etc/rhsm/rhsm.conf
+
+  See <filter>.
+*)
+
+module Rhsm =
+  autoload xfm
+
+let lns     = IniFile.lns_loose_multiline
```
Comment (raphink) [inline lenses/rhsm.aug]: As said on IRC, I'd rather use `IniFile.lns_loose*` lenses for guessing work, not specific lenses.

Comment (domcleal) [inline lenses/rhsm.aug]: Thanks for the comment, I've updated it to be precise about what Python's ConfigParser supports.

**Comment by raphink** [comment]
Looking good, want to merge it?

**Comment by domcleal** [comment]
Thanks for reviewing, abdb9fb.

**Pull request closed without merging**